tests/io/v3/vm/agent_groups/test_agent_groups_schema.py

```diff
@@ -0,0 +1,41 @@
+'''
+Testing the AgentGroups schemas
+'''
+from tenable.io.v3.vm.agent_groups.schema import AgentGroupsBaseSchema
+
+
+def test_agent_groups_schema_with_name():
+    '''
+    Test the agent_groups schema with name
+    '''
+    name: str = 'sample name'
+    payload = {
+        'name': name
+    }
+    test_resp = {
+        'name': name
+    }
+    schema = AgentGroupsBaseSchema(only=['name'])
```

Comment:
According to me, we don't need to use only keyword in the complete Agent Group API.

Reply:
Updated.